### DISCUSSION TITLE
[Q&A] 请问，超出限制后怎么进行持续进行讨论

### DISCUSSION THREAD

**Discussion by yikuaidao** [lobehub/lobehub#988] [General Question] [answered]
![微信图片编辑_20240109134821](https://github.com/lobehub/lobe-chat/assets/56807964/fce81e95-a99f-497e-81d5-9874b7fef63f)

**Comment by arvinxx** [comment, marked as answer]
几种思路：

1. 一种是换更大上下文的模型（gpt-4-1106-preivew）；
2. 添加上下文条数限制，以减少进入上下文窗口的消息；
<img width="296" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/f1e62dc3-47ea-4845-b536-c087b36e119c">


还有一种方案是我们之前想做，但是还没想好怎么做：
- https://github.com/lobehub/lobe-chat/issues/34

**Reply by yikuaidao** [reply]
感谢，这是一个很棒的产品，期待能接入更多的模型！
